src/block_implementations/Eos/EosFaderBankBlock.h

```diff
@@ -97,7 +102,9 @@ protected slots:
     QVector<bool> m_feedbackInvalid;
 
     BoolAttribute m_catchFaders;
+    BoolAttribute m_pageChangeMode;
```

Comment:
Please add an explanation in the helpText what the pageChangeMode does and how to set it.